packages/account/src/Configs/trading-assessment-config.js

```diff
@@ -0,0 +1,305 @@
+                    },
+                    {
+                        text: localize('1 - 2 years'),
+                        value: '1 - 2 years',
+                    },
+                    {
+                        text: localize('Over 3 years'),
+                        value: 'Over 3 years',
+                    },
+                ],
+            },
+            {
+                question_text: localize('How many CFD trades have you placed in the past 12 months?'),
+                field_type: 'dropdown',
+                form_control: 'cfd_frequency',
+                answer_options: [
+                    {
+                        text: localize('None'),
+                        value: 'No transactions in the past 12 months',
+                    },
+                    {
+                        text: '1 - 5',
+                        value: '1 - 5 transactions in the past 12 months',
+                    },
+                    {
+                        text: '6 - 10',
+                        value: '6 - 10 transactions in the past 12 months',
+                    },
+                    {
+                        text: '11 - 39',
+                        value: '11 - 39 transactions in the past 12 months',
+                    },
+                    {
+                        text: localize('40 or more'),
+                        value: '40 transactions or more in the past 12 months',
+                    },
+                ],
+            },
+            {
+                question_text: localize('How much experience do you have with other financial instruments?'),
+                field_type: 'dropdown',
+                form_control: 'trading_experience_financial_instruments',
+                answer_options: [
+                    {
+                        text: localize('No experience'),
+                        value: 'No experience',
+                    },
+                    {
+                        text: localize('Less than a year'),
+                        value: 'Less than a year',
+                    },
+                    {
+                        text: localize('1 - 2 years'),
+                        value: '1 - 2 years',
+                    },
+                    {
+                        text: localize('Over 3 years'),
+                        value: 'Over 3 years',
+                    },
+                ],
+            },
+            {
+                question_text: localize(
+                    'How many trades have you placed with other financial instruments in the past 12 months?'
+                ),
+                form_control: 'trading_frequency_financial_instruments',
+                field_type: 'dropdown',
+                answer_options: [
+                    {
+                        text: localize('None'),
+                        value: 'No transactions in the past 12 months',
+                    },
+                    {
+                        text: '1 - 5',
```

Comment:
Here the value and text do not match and value contains more details than the text shown to user? Shouldn't it be an opposite?

Reply:
The value that is to be sent has to be the whole sentence. The value is part of an enum that BE is checking for. 
The text shown is  as per the Figma design